src/components/overlay/overlay.tsx

```diff
@@ -85,7 +85,7 @@ class Overlay extends Component<OverlayProps, any> {
     } else {
       // Remove the overlay-active class only when overlay portal has a single child
       const overlayPortalEl = getOverlayPortalElement(this.props.player);
-      if (overlayPortalEl!.childElementCount <= 1) {
+      if (overlayPortalEl && overlayPortalEl?.childElementCount <= 1) {
```

Comment:
you don't need to check if overlayPoratlE1 exist anymore, since using ? operator

Reply:
You're right. Removed.